src/forms/Predictions.js

```diff
@@ -171,17 +171,23 @@ const TabbedVisualizations = ({ client }) => {
         { menuItem: 'Changes Chart', render: () => {return <Tab.Pane><Chart client={ client } /></Tab.Pane>;} },
         {
           menuItem: 'Stacked Incomes',
-          render:   () => {return <Tab.Pane><GraphHolder
-            client={ client }
-            Graph={ GrossGraph } />
-          </Tab.Pane>;}, 
+          render:   () => { return (
+            <Tab.Pane>
+              <GraphHolder
+                client={ client }
+                Graph={ GrossGraph } />
+            </Tab.Pane>
+          );},
```

Comment:
I'd put the `return` on the line after the braces, personally. Not a dealbreaker.